docs/modules/sql/pages/mapping-to-jdbc.adoc

```diff
@@ -8,6 +8,16 @@
 
 The JDBC connector allows you to connect to any database that supports the JDBC interface.
 
+== Supported Databases
+
+The JDBC connector officially supports the following database management systems:
```

Comment:
```suggestion
The JDBC connector supports the following database management systems:
```

Reply:
@rebekah-lawrence I think we need a bit of help here how to formulate this.

These 4 are the DBMS we know it works with and we support them. There are many other DBMS that provide JDBC driver, but we haven't tested those. But they still may work fine, maybe for some subset of commands. 

What should we do? Maybe add below

> Other database management systems with JDBC interface may still work, but support from Hazelcast is not provided.

or something like that?

Reply:
I'd make it a note, perhaps something like:

NOTE: You can use other database management systems with a JDBC interface. However, Hazelcast offers no guarantees or support for these systems.